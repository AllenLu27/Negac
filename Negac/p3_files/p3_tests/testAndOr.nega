int main() {
	int a;
	a = false && false || false;
}
