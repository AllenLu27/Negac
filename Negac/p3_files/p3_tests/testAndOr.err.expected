FATAL [3,4]: Invalid assignment operation
Semantic analysis failed
